crates/cli/index.js

```diff
@@ -0,0 +1,3 @@
+const { run } = require('./index.node')
```

Comment:
Should we publish it as standalone binary?

Reply:
it exists only for test convenience